nixos/modules/services/web-apps/miniflux.nix

```
@@ -103,45 +105,27 @@ in
         ++ lib.optionals cfg.createDatabaseLocally [ "postgresql.service" "miniflux-dbsetup.service" ];
 
       serviceConfig = {
-        Type = "notify";
-        ExecStart = lib.getExe cfg.package;
+        ExecStart = [ "" (lib.getExe cfg.package) ];
+        EnvironmentFile = lib.optional (cfg.adminCredentialsFile != null) [ "" cfg.adminCredentialsFile ];
```

```suggestion
        EnvironmentFile = lib.optionals (cfg.adminCredentialsFile != null) [ "" cfg.adminCredentialsFile ];
```

This should fix the tests.